docs/rules/arrow-parens.md

```diff
@@ -176,4 +177,44 @@ a.then(foo => { if (true) {}; });
 (a = 10) => a;
 ([a, b]) => a;
 ({a, b}) => a;
-```
\ No newline at end of file
+```
+
+### requireForBlockBody
+
+Examples of incorrect code for the { "requireForBlockBody": true } option:
```

Comment:
Oops, it looks like this line and the corresponding "correct" line lost some formatting when you copied them from my previous comment. The words "incorrect" and "correct" need double asterisks, and the code needs backticks:

```md
Examples of **incorrect** code for the `{ "requireForBlockBody": true }` option:
```